os/board/artik05x/Kconfig

```diff
@@ -165,6 +165,14 @@ config ARTIK05X_AUTOMOUNT_ROMFS_MOUNTPOINT
 		Specifies the mount point where romfs file system
 		will be mounted at.
 
+config SVR_DB_SECURESTORAGE
```

Comment:
I think this configuration is better in ST_Things kconfig.

Reply:
This PR is to sync with DS git.

Reply:
Hi @HONGCHAEHEE, this configuration also depends on CONFIG_TLS_WITH_SSS, because it uses mbedtls APIs defined in sss_storage.c, and sss_storage is complied and registered as lib only when CONFIG_TLS_WITH_SSS is defined.